consul/src/main/java/org/secondbase/consul/ConsulModule.java

```diff
@@ -0,0 +1,190 @@
+package org.secondbase.consul;
+
+import com.google.common.net.HostAndPort;
+import com.orbitz.consul.Consul;
+import com.orbitz.consul.model.agent.Registration;
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.SecondBaseException;
+import org.secondbase.core.config.SecondBaseModule;
+
+/**
+ * Consul module for SecondBase. Handles registration of services in Consul.
```

Comment:
Use "consul" instead here. I have a feeling it's mostly used with a lowercase. Super important, i know.